src/Marten/Linq/Filters/FullTextWhereFragment.cs

```diff
@@ -16,7 +16,7 @@ internal class FullTextWhereFragment: ISqlFragment
         private readonly FullTextSearchFunction _searchFunction;
         private readonly string _searchTerm;
 
-        private string Sql => $"to_tsvector(:argRegConfig::regconfig, {_dataConfig}) @@ {_searchFunction}(:argRegConfig::regconfig, :argSearchTerm)";
+        private string Sql => $"to_tsvector('{_regConfig}'::regconfig, {_dataConfig}) @@ {_searchFunction}('{_regConfig}'::regconfig, '{_searchTerm}')";
```

Comment:
recommend doing a find/replace of `'` with `''` for safety

Reply:
Ok will do, is there a extension / pre written function you have already for this type of thing?

Reply:
doesnt look like there is one in npgsql. Best thing I can find based on [this doco](https://www.postgresql.org/docs/8.3/sql-syntax-lexical.html#SQL-SYNTAX-STRINGS) is to turn the string into an E string then find replace `'` with `''` and `\` with `\\`.